docs/source/index.rst

```diff
@@ -24,6 +24,7 @@ Contents
    building
    measurement
    reconstruction
+   trainable_reconstruction
```

Comment:
wondering if we should rather have this inside reconstruction? And have sections for: "classical", "unrolled", "plug-and-play", "deep learning", etc

Reply:
Yes, it seems better. However, I am not quite sure how to do it properly...